addons/mail/models/mail_mail.py

```diff
@@ -511,11 +540,13 @@ def _send(self, auto_commit=False, raise_exception=False, smtp_session=None):
                 # TDE note: could be great to pre-detect missing to/cc and skip sending it
                 # to go directly to failed state update
                 for email in email_list:
+                    body_with_embedded_attachment_links = email.get('body') if not attachments_links \
+                        else tools.append_content_to_html(email.get('body'), attachments_links, plaintext=False)
```

Comment:
a bit more readable imo
```suggestion
                    if attachments_links:
                        email['body'] =   tools.append_content_to_html(email.get('body'), attachments_links, plaintext=False)
```

Reply:
Yes good idea, thanks